config/lava/chromeos/chromeos-flash-template.jinja2

```diff
@@ -50,35 +50,12 @@ actions:
         run:
           steps:
             - ping -c 1 172.17.0.1
-            - lava-test-case wget --shell  "wget http://172.17.0.1/chromeos/{{ chromeos_image }}"
```

Comment:
The reason for using `172.17.0.1` is to flash proprietary Chrome OS images that aren't stored on storage.chromeos.kernelci.org.  I don't think we've completely confirmed yet whether we'll be able to use the Chromium OS builds and not the ones with proprietary components, so we probably should keep this for the time being.

It could be done with a different PR later on, and this PR could be just about using SSH rather than the serial console.

Reply:
With my PR it is still possible to fetch private image, just instead of using hardcoded http://172.17.0.1/chromeos/ we specify full url.
It can be: 
chromeos_image: 'http://172.17.0.1/chromeos/chromiumos_test_image-R94-14150.19.0-octopus.bin'
or
chromeos_image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/crostest007/chromiumos-hatch/amd64/chromiumos_test_image.bin.gz'

I am trying to make templates such way, that we can avoid changing them later. 
But then i need to find way to remove hardcoded filename chromiumos_test_image.bin.gz, or internal image can be moved to something like 'http://172.17.0.1/chromeos/chromiumos_test_image-R94-14150.19.0-octopus/chromiumos_test_image.bin.gz'

Reply:
OK, I see.  Maybe use `basename` to get the file name from the URL, or some Python code to handle this with `urllib`?

Reply:
I will try to figure that out, so it will work with any url. Filename part is easiest , i can use -O option of wget, but about compressed and uncompressed part it is a bit more tricky

Reply:
Sorted out too. Trick is simple, i just wget -o chromiumos_test_image.bin.gz, then attempt to unpack and if it fails, i assume it is not compressed and just rename it.

gzip -d chromiumos_test_image.bin.gz || mv chromiumos_test_image.bin.gz chromiumos_test_image.bin

Compressed:
https://lava.collabora.co.uk/scheduler/job/6187231
Uncompessed (manually moved to my host)
https://lava.collabora.co.uk/scheduler/job/6187470